CONTRIBUTING.md

```diff
@@ -24,15 +24,18 @@ The submitted code should adhere to the following:
 
 ### Do's
 
-* For every pull request, make sure all commits are consolidated into one commit before
-  sending the pull request.
+* Keep pull requests focused on addressing one issue or feature. If multiple separate issues
+  are fixed by a pull request, prefer commiting the fixes under multiple commits, one for each issue.
+* For every pull request, make sure all commits for a related issue or feature are consolidated
+  into a single commit before sending the pull request.
 * Before submitting any code changes, please ensure that the [Unit Tests](https://github.com/google/EarlGrey/tree/master/Tests/UnitTests)
```

Comment:
it's easier to review pull requests as individual commits and then have GitHub auto squash them via the new squash button.

https://github.com/blog/2141-squash-your-commits